Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Installation stage
+# Offical Node.js Alpine base image
+FROM node:lts-alpine AS base
+
+# Set working directory
+WORKDIR /base
+
+# Copy package.json and package-lock.json for utilising Docker cache 
+# to save re-installing dependencies if unchanged
+COPY package*.json yarn.lock ./ 
+
+# Install dependencies only needed
+RUN yarn install --production --frozen-lockfile
+
+# Copy installed dependencies to current working directory
+COPY . .
+
+# Build stage
+FROM base AS builder
+ENV NODE_ENV=production
```

Comment:
FYI, I've tested that currently the `ENV` isn't needed. We can remove for now.

Reply:
I've set it in case we create `docker-compose` in further development.

Reply:
If it's for future implementation then lets remove ir for now since it's unused. 😇